pkg/kapp/cmd/app/logs.go

```diff
@@ -71,7 +71,14 @@ func (o *LogsOptions) Run() error {
 		supportObjs.IdentifiedResources.PodResources(labelSelector),
 	}
 
-	logsView := ctllogs.NewView(logOpts, podWatcher, supportObjs.CoreClient, o.ui)
+	contFilter := func(pod corev1.Pod) []string {
+		if len(o.LogsFlags.ContainerNames) > 0 {
```

Comment:
dont think we need to check > 0, can just return.

Reply:
Too paranoid :)